properties/config.json

```diff
@@ -118,7 +126,11 @@
       },
       "EPIC": false,
       "cBio": false,
+      "linkToHomePage": "https://matchminer.dfci.harvard.edu",
```

Comment:
Do we want these links in the public site? I see the request access link to google form was already there? Do we want to strip that?

Reply:
These links were already there, and they don't apply to the setup script, which installs its own configuration in place of this one. So this configuration is just to keep the code working outside of the setup script, within DFCI.